### PR TITLE
Unify code comments style.

### PR DESCRIPTION
**Pull request by Skyge**
Fixes #

Unify code comments style with Doxygen-style tags. Because mixing two style is really weird.
Such as in a contract it likes this :
```
/**
*@dev Total number of tokens in existence
*/
```
so, I update style to like this:
```
/**
 *@dev Total number of tokens in existence
 */
```
I have discussed this with @frangio about two days ago.

### CODE REVIEW

File: contracts/mocks/ERC165/ERC165InterfacesSupported.sol [hunk 1]
```diff
@@ -4,9 +4,8 @@ import "../../introspection/IERC165.sol";
 
 /**
  * https://github.com/ethereum/EIPs/blob/master/EIPS/eip-214.md#specification
- * > Any attempts to make state-changing operations inside an execution instance with STATIC set to true will instead throw an exception.
- * > These operations include [...], LOG0, LOG1, LOG2, [...]
- *
+ * Any attempts to make state-changing operations inside an execution instance with STATIC set to true
```
[the remark this answers is not in the document]
Comment (Skyge) [reply contracts/mocks/ERC165/ERC165InterfacesSupported.sol]: ohhh, so sorry for that, I did not make sense of your original intention for this, I will make changes for it later.

File: contracts/token/ERC721/ERC721.sol [hunk 2]
```diff
@@ -31,7 +31,7 @@ contract ERC721 is ERC165, IERC721 {
     mapping (address => mapping (address => bool)) private _operatorApprovals;
 
     bytes4 private constant _InterfaceId_ERC721 = 0x80ac58cd;
-    /*
+    /**
```
[the remark this answers is not in the document]
Comment (Skyge) [reply contracts/token/ERC721/ERC721.sol]: As for this change, I actually thought for a while, but after that, I found in `contracts/introspection/ERC165.sol`, `contracts/mocks/ERC165/ERC165InterfacesSupported.sol`, `contracts/introspection/ERC165Checker.sol`, `contracts/token/ERC721/ERC721Metadata.sol`, `contracts/token/ERC721/ERC721Enumerable.sol`, they all use `/** /` for comments, so I change this one to unify, so what should I do now, I should just left this as before, or I should change other file?

File: contracts/mocks/ERC165/ERC165InterfacesSupported.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (Skyge) [reply contracts/mocks/ERC165/ERC165InterfacesSupported.sol]: @frangio OK, I have changed it back.

**Comment by frangio** [comment]
Sorry for the delay @Skyge. Thanks!

**Pull request merged**